src/matter.rs

```diff
@@ -12,7 +12,7 @@ enum Part {
 /// Coupled with an [`Engine`](crate::engine::Engine) of choice, `Matter` stores delimiter(s) and
 /// handles parsing.
 pub struct Matter<T: Engine> {
-    pub delimiter: String,
+    pub delimiter: [String; 2],
```

Comment:
Can you add a `pub close_delimiter: Option<String>` instead of changing it directly? It is better to make no breaking changes.

Reply:
Sure, I'll add it later today 👍